kube_log_watcher/templates/scalyr.json.jinja2

```diff
@@ -2,10 +2,10 @@
     "api_key": "{{ scalyr_key }}",
     "max_log_offset_size": 100000000,
```

Comment:
200000000

Reply:
Also `compression_level: 9`

Reply:
BTW I really doubt about `compression_level: 9`, because usually it improves compression a little, but takes much more CPU. In our case if we're hitting CPU limits, I think this is wrong setup.

Reply:
We want to test `compression_level: 9` in a different PR to see how much the CPU usage changes. If it is to much we won't apply it.

Reply:
Setting the `max_log_offset_size` to 200000000 is just a recommendation by scalyr but doesn't match our setup. Logfiles are rotated after 50 MB and at maximum 2 files are kept.